packages/cli/bin/merkur.mjs

```diff
@@ -28,6 +31,36 @@ const inspectOption = new Option('--inspect', 'Debugging widget server');
 const verboseOption = new Option('--verbose', 'Verbose mode which show debug information.');
 const analyzeOption = new Option('--analyze', 'Analyze bundle.');
 
+let customCommands = [];
+
+// Extend commands from directory
+const extendCommandsFromDir = async commandsDir => {
+  customCommands = customCommands.concat(
+    fs.readdirSync(commandsDir)
+      .map(command => ({ dir: commandsDir, command }))
+      .filter(
+        ({ command, dir }) => !customCommands.some(({ command: existingCommand }) => existingCommand === command) &&
+        fs.statSync(path.join(dir, command)).isFile() &&
+        (command.endsWith('.js') || command.endsWith('.mjs'))
+      )
+  );
+};
+
+// Get merkur custom commands
+const merkurDir = path.resolve(process.cwd(), 'node_modules/@merkur');
+if (fs.existsSync(merkurDir)) {
+    let dirs = [];
+    dirs = fs.readdirSync(merkurDir)
+      .map(dir => ({ dir }))
+
+    for (const { dir } of dirs) {
```

Comment:
This mapping feels a bit odd, you are changeing the structure from `dirs = ['dir1', 'dir2']` to `dirs = [{ dir: 'dir1' }, { dir: 'dir2' }]` and right below this line you are again parsing the `dir` key from each object. What is the point of changing this structure?

```suggestion
    dirs = fs.readdirSync(merkurDir);

    for (const dir of dirs) {
```

Reply:
We don't need to change the structure. Removing `dir` key.